sdks/wasm/Mono.WebAssembly.DebuggerProxy/DevToolsHelper.cs

```diff
@@ -109,14 +110,10 @@ internal struct Result {
 			if (result != null && error != null)
 				throw new ArgumentException ($"Both {nameof(result)} and {nameof(error)} arguments cannot be non-null.");
 
-			bool resultHasError = String.Compare ((result? ["result"] as JObject)? ["subtype"]?. Value<string> (), "error") == 0;
-			if (result != null && resultHasError) {
-				this.Value = null;
-				this.Error = result;
-			} else {
-				this.Value = result;
-				this.Error = error;
-			}
+			ResultHasError = result? ["exceptionDetails"] != null || String.Compare ((result? ["result"] as JObject)? ["subtype"]?. Value<string> (), "error") == 0;
+
+			this.Value = result;
+			this.Error = error;
 		}
 
 		public static Result FromJson (JObject obj)
```

Comment:
Here we are looking for `result`, and an `error` field in the object. Where is this `error` field mentioned in the protocol? I think it would help to differentiate, or figure out if we need only one bool property to indicate error/failure.

Reply:
I tried to avoid making any unrelated changes - and since both the `result` and the `error` are parameters that are being passed in by the consumer of this struct, I didn't want to change their semantics.  Similarly, the `ResultHasError` is meant to reflect the old behavior.